ingestion/sources/postgresql/pg-cdc.mdx

```diff
@@ -206,6 +206,99 @@ RisingWave does not support directly creating tables from PostgreSQL composite t
 | MACADDR                                              | CHARACTER VARYING\*                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
 | MACADDR8                                             | CHARACTER VARYING\*                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
 
+### Support for PostgreSQL TOAST
+
+<Note>
+Added in v2.6.0.
+</Note>
+
+RisingWave supports [TOASTed (The Oversized-Attribute Storage Technique) data](https://www.postgresql.org/docs/current/storage-toast.html) from PostgreSQL when using the CDC connector. This ensures that even columns with very large values, such as long text or large JSON objects, are ingested completely and accurately during both the initial backfill and incremental changes.
```

Comment:
Please emphasize:
1. We support updates for the types we mentioned.
2. For types not mentioned but that may still trigger TOAST (primarily simple one-dimensional arrays, which we believe have a very low probability of triggering TOAST), we currently do not support them. Please refer to the [issue](https://github.com/risingwavelabs/risingwave/issues/22916)  for details, you can put the issue link in doc.